shipping_addresses.go

```diff
@@ -0,0 +1,77 @@
+package recurly
+
+import (
+	"encoding/xml"
+)
+
+// ShippingAddress represents a shipping address
+type ShippingAddress struct {
+	XMLName       xml.Name `xml:"shipping_address"`
+	AccountCode   string   `xml:"account,omitempty"`
+	Subscriptions string   `xml:"subscriptions,omitempty"`
```

Comment:
This will currently return the shipping address ID because the `hrefString` in this response is as follows:
```
<subscriptions href="https://your-subdomain.recurly.com/v2/accounts/1/shipping_addresses/2438622711411416831/subscriptions"/>
```
Let's omit this line here and add a new endpoint `GetSubscriptions` for retrieving the subscriptions for a shipping address, which returns an array of subscriptions (I ran a quick test in Postman to see what this result looked like). You may be able to use the existing `Subscription` struct but you'll have to check the fields.